index.bs

```diff
@@ -210,48 +210,55 @@ XRHand {#xrhand-interface}
 ------
 
 <pre class="idl">
+enum XRHandJoint {
+  "wrist",
+
+  "thumb-palm",
+  "thumb-base",
+  "thumb-upper",
+  "thumb-tip",
+
+  "index-finger-palm",
+  "index-finger-base",
+  "index-finger-lower",
+  "index-finger-upper",
+  "index-finger-tip",
+
+  "middle-finger-palm",
+  "middle-finger-base",
+  "middle-finger-lower",
+  "middle-finger-upper",
+  "middle-finger-tip",
+
+  "ring-finger-palm",
+  "ring-finger-base",
+  "ring-finger-lower",
+  "ring-finger-upper",
+  "ring-finger-tip",
+
+  "pinky-finger-palm",
+  "pinky-finger-base",
+  "pinky-finger-lower",
+  "pinky-finger-upper",
+  "pinky-finger-tip",
+};
+
 [Exposed=Window]
 interface XRHand {
     iterable&lt;XRJointSpace>;
```

Comment:
```suggestion
    iterable&lt;XRJointName, XRJointSpace>;
```

This might work well. What this means is:
 - `for names in hand` gives you the names
 - `for joints of hand` gives you the joint spaces
 - `fillPoses()` will take `hand.values()` instead of `hand`

The main benefit is that the order of joints is now accessible from code; so if you use `fillPoses()` or something you have a reference order to use. Since this ordering is fixed people can also hardcode it, but it's nicer if they don't want to rely on it.

Reply:
> for names in hand gives you the names / for joints of hand gives you the joint spaces

Not quite. You'd use `for (let name of hand.keys())` and `for (let joint of hand.values())`. If you didn't use `keys()`/`values()`, you'd get a 2-element array, i.e. `for (let [name, joint] of hand.values())`.

Reply:
@domenic if we were to replace `iterable<XRJointSpace>` to `iterable<XRJointName, XRJointSpace>' can we still define the order?

Reply:
Yep, defining the order is still possible, and in fact kinda mandatory for interoperability.

Reply:
Great! I didn't know that :-)
I will update the IDL since it makes it simpler